src/main/java/com/adepuu/exercises/session7/User.java

```diff
@@ -0,0 +1,111 @@
+package com.adepuu.exercises.session7;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.UUID;
+
+public class User {
+  private String userId;
+
+  private String username;
+
+  private String password;
+
+  private boolean isLogin;
+
+  private List<HashMap<String, String>> users = new ArrayList<>();
```

Comment:
for this approach, i think just use `HashMap` without the `List` just because the HashMap can also be iterated (using loop or `forEach`)

Reply:
or make it `HashMap<String, User> users`